### ISSUE TITLE
SVF highpass mode letting through sound when cutoff is high

### ISSUE THREAD

**Issue by markwheeler** [supercollider/sc3-plugins#248]
I'm seeing some odd behavior with the SVF ugen (version included in SC 3.10). When using it as a high-pass filter it works as expected when the cutoff is below ~10500, but as it moves above that, lower frequencies start to leak through again. Here's a tiny code snippet to try:

`{SVF.ar(WhiteNoise.ar * 0.5, MouseX.kr(10, 20000).poll(4), 0, 0, 0, 1)}.play;`

**Comment by snappizz** [comment]
yeah, SVF is just some free code copied off musicdsp.org. i'm not an expert here but it looks to be a naive chamberlin-type SVF, not one of the nice modern trapezoidal ones.

also it doesn't even interpolate its control-rate inputs. unfortunately this is the sort of code quality one should expect from sc3-plugins.
